docs/user/index.rst

```diff
@@ -159,7 +160,7 @@ Need to get something specific done? These guides provide step-by-step instructi
 Reference
 ---------
 
-Need to know how something works? Our references provide the details:
+Need to know how something works? Here are a few of the most important reference docs:
```

Comment:
```suggestion
Need to know how something works? 

Here are a few of the most important reference docs:
```

Seems like this would keep the structure the same?